app/views/users/_header.html.erb

```diff
@@ -28,6 +28,10 @@
           </li>
         <% end %>
       </ul>
+      <form class="d-flex" action="/search">
+        <input class="form-control me-2" name="q" type="search" placeholder="Search" aria-label="Search">
```

Comment:
I know I'm not the one reviewing this, but I was just being nosy and having a look and I think this should be `action="posts/search"`?